src/generators/networkCreator.js

```diff
@@ -70,6 +70,12 @@ export function generateResourcesRemote(config) {
     writeFile(permissionedNodesLocation, nodesWithSubnetReplaced)
     copyDirectory(remoteOutputDir, configDir)
   }
+  if (config.network.splunk) {
+    copyDirectory(joinPath(libRootDir(), 'lib', 'splunk'), joinPath(remoteOutputDir, 'splunk'))
+  }
+  if (config.network.txGenerate) {
+    copyDirectory(joinPath(libRootDir(), 'lib', 'splunk', 'contracts'), joinPath(remoteOutputDir, 'contracts'))
```

Comment:
the contract_config.json file isn't copied here (it _is_ copied if you have splunk enabled because it's in that splunk directory). Also the splunk directory copy includes the contracts folder a second time at splunk/contracts. Maybe reorganize the directories a bit to make these copies easier. And make sure the docker volumes are pointing to the right folders.

Reply:
this was removed